src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -2220,12 +2222,28 @@ NetworkOPsImp::pubValidation(std::shared_ptr<STValidation> const& val)
             reserveIncXRP && reserveIncXRP->native())
             jvObj[jss::reserve_inc] = reserveIncXRP->xrp().jsonClipped();
 
+        // NOTE Use MultiApiJson to publish two slightly different JSON objects
+        // for consumers supporting different API versions
+        MultiApiJson multiObj{jvObj};
+        visit<RPC::apiMinimumSupportedVersion, RPC::apiMaximumValidVersion>(
```

Comment:
This PR is meant to make the change trivial switching between API version 3 and version 2, which is why it looks like this. If this is indeed released for API version 2 then I have another PR lined up to clean this up, so we simply modify one version of `Json::Value` rather than `visit` here - i.e. simplify accordingly